libr/anal/p/anal_arm_cs.c

```diff
@@ -228,7 +228,7 @@ static void arm64math(RAnal *a, RAnalOp *op, ut64 addr, const ut8 *buf, int len,
 				}
 			}
 		}
-		
+
```

Comment:
kill that line, with fire if needed, its empty, so it shouldnt burn for much time